config/jobs/kubernetes-sigs/kind/kind.yaml

```diff
@@ -212,46 +236,3 @@ periodics:
     description: Runs conformance tests using kubetest against latest kubernetes master with an IPv6 kubernetes-in-docker cluster, skipping [Serial] tests
     testgrid-alert-email: bentheelder@google.com,antonio.ojea.garcia@gmail.com
     testgrid-num-columns-recent: '3'
-# conformance test against kubernetes release-1.15 branch with `kind`
```

Comment:
I think this is a leftover, is the only release is being tested here, the others are testing master only

Reply:
I don't think so, IIRC in 1.16 spiffxp switched some around.

Reply:
anyway, it went out of support, do we want to keep testing it?

Reply:
we should leave it in kind presubmit. removing here is OK.